testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenode.java

```diff
@@ -1,37 +0,0 @@
-/*
```

Comment:
Could you remove "singlenode" from `trino-product-tests/README.md`?

Reply:
Thanks for pointing this. Have updated the README.md